pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -192,8 +430,12 @@ func (s *Service) marshalAndSend(data interface{}, errorMsg, destination string)
 	return s.outboundTransport.Send(string(jsonString), destination)
 }
 
+func (s *Service) generateRandomID() string {
```

Comment:
I don't see a reference to s in this method.

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/6dc9c7de378434ea27520364556f649ccf434c5f..36d8a92bfaa82b830e44a047521b376920c82c72#diff-f761c91e753eba1c42a68076df06ce12R437).